docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3'
+
+services:
+  db:
+    environment:
+      POSTGRES_DB: imagetagger
+      POSTGRES_USER: imagetagger
+      POSTGRES_PASSWORD: imagetagger
```

Comment:
This does not seem production-ready with an (unsecure) static password. It should rather be set from the environment (cf. https://docs.docker.com/engine/reference/commandline/run/#set-environment-variables--e-env-env-file).

Furthermore, I do not like the idea of running a database inside a container (cf. https://encrypted.google.com/search?hl=en&q=database%20inside%20docker).